app/Http/Controllers/Auth/RegisterController.php

```diff
@@ -65,9 +65,10 @@ protected function validator(array $data)
     protected function create(array $data)
     {
         $user = DB::transaction(function () use ($data) {
+            $lowerUserEmail = strtolower($data['student_number']);
```

Comment:
Replace this with in-line statement.

`$data['student_number'] = strtolower($data['student_number']);`